superset/utils/pandas_postprocessing.py

```diff
@@ -590,6 +591,9 @@ def _prophet_fit_and_predict(  # pylint: disable=too-many-arguments
     Fit a prophet model and return a DataFrame with predicted results.
     """
     try:
+        prophet_logger = logging.getLogger("fbprophet.plot")
+
+        prophet_logger.setLevel(logging.CRITICAL)
```

Comment:
I don't feel strongly about it either, but should we crank the level back up after the import?

Reply:
Good idea. Updated and verified that initial error stays unlogged.